tests/integration/release_test.go

```diff
@@ -81,6 +81,21 @@ func TestViewReleasesNoLogin(t *testing.T) {
 	MakeRequest(t, req, http.StatusOK)
 }
 
+func TestViewLatestRelease(t *testing.T) {
+	defer tests.PrepareTestEnv(t)()
+
+	session := loginUser(t, "user2")
+	req := NewRequest(t, "GET", "/user2/repo1/releases/latest")
+	session.MakeRequest(t, req, http.StatusOK)
+}
+
+func TestViewLatestReleaseNoLogin(t *testing.T) {
```

Comment:
I can see one to two more test cases:
1. non-public releases are omitted when not logged in. Potentially overkill.
2. draft releases are not returned (even for site admins/ users with write rights).